### PR TITLE
Add colouring for steps status 🎨 

### PR DESCRIPTION
**Pull request by chmouel**
# Changes

Steps status is coloured now 🌈 🎨 

image

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Regenerate the manpages and docs with `make docs` and `make man` if needed.
- [ ] Run the code checkers with `make check`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

File: pkg/formatted/k8s.go [hunk 1]
```diff
@@ -24,10 +24,13 @@ var ConditionColor = map[string]color.Attribute{
 	"Failed":    color.FgHiRed,
 	"Succeeded": color.FgHiGreen,
 	"Running":   color.FgHiBlue,
-	"Cancelled": color.FgHiMagenta,
+	"Cancelled": color.FgHiWhite,
+	"Completed": color.FgHiMagenta,
+	"Pending":   color.FgHiYellow,
```
Comment (danielhelfand) [inline pkg/formatted/k8s.go]: Would it be better to do Yellow for Cancelled and White for Pending? Pending is very unlikely to be seen as `PodInitializing` status actually happens really quick. Maybe it makes more sense to use color to call attention to cancelation?

Comment (chmouel) [reply pkg/formatted/k8s.go]: I have no preference :D but your suggestions sounds good!

**Comment by chmouel** [comment]
How come the tests pass on my laptop 🤔 

image

**Pull request merged**